chart/values.yaml

```diff
@@ -5,6 +5,16 @@ image:
   repo: openebs
   # -- Release tag for our images
   tag: develop
+  repoTags:
+    # Note: Below image tag configuration is optional and typically should never be
+    #       used. Setting specific image tags for the different repositories proves useful 
+    #       for some integration testing scenarios. Use the 'tag' option above to set
+    #       released/pre-release container image tags.
```

Comment:
nit: Might as well call it 'release' instead of 'released'.

Reply:
Changed to release